spaceship/lib/spaceship/connect_api/tunes/tunes.rb

```diff
@@ -1278,6 +1313,260 @@ def get_review_rejection(filter: {}, includes: nil)
+          body = {
+            data: {
+              type: "appClipDefaultExperiences",
+              relationships: {
+                appClip: {
+                  data: {
+                    type: 'appClips',
+                    id: app_clip_id
+                  }
+                },
+                releaseWithAppStoreVersion: {
+                  data: {
+                    type: "appStoreVersions",
+                    id: app_store_version_id
+                  }
+                }
+              }
+            }
+          }
+
+          body[:data][:attributes] = attributes unless attributes.nil?
+
+          unless template_default_experience_id.nil?
+            body[:data][:relationships][:appClipDefaultExperienceTemplate] = {
+              data: {
+                id: template_default_experience_id,
+                type: 'appClipDefaultExperiences'
+              }
+            }
+          end
+
+          tunes_request_client.post("#{Version::V1}/appClipDefaultExperiences", body)
+        end
+
+        # https://developer.apple.com/documentation/appstoreconnectapi/modify_a_default_app_clip_experience
+        def patch_app_clip_default_experience(default_experience_id: nil, attributes: {})
+          body = {
+            data: {
+              id: default_experience_id,
+              type: "appClipDefaultExperiences",
+              attributes: attributes
+            }
+          }
+
+          tunes_request_client.patch("#{Version::V1}/appClipDefaultExperiences/#{default_experience_id}", body)
+        end
+
+        #
+        # appClipDefaultExperienceLocalizations
+        #
+
+        def get_app_clip_default_experience_header_image(app_clip_default_experience_localization_id:, filter: {}, includes: nil, limit: nil, sort: nil)
+          params = tunes_request_client.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
+          tunes_request_client.get("#{Version::V1}/appClipDefaultExperienceLocalizations/#{app_clip_default_experience_localization_id}/relationships/appClipHeaderImage", params)
+        end
+
+        # https://developer.apple.com/documentation/appstoreconnectapi/create_the_localized_metadata_for_a_default_app_clip_experience
+        def post_app_clip_default_experience_localization(default_experience_id: nil, attributes: {})
+          body = {
+            data: {
+              type: "appClipDefaultExperienceLocalizations",
+              attributes: attributes,
+              relationships: {
+                appClipDefaultExperience: {
+                  data: {
+                    type: 'appClipDefaultExperiences',
+                    id: default_experience_id
+                  }
+                }
+              }
+            }
+          }
+
+          tunes_request_client.post("appClipDefaultExperienceLocalizations", body)
```

Comment:
Missing `#{Version::V1}/` from API path, so when creating App Clip localizations on a new release page it fails with `Spaceship::UnexpectedResponse` exceptions.